tests/unit/test_charm_2.py

```diff
@@ -630,3 +698,101 @@ def test_plugin_reconciliation(self):
             set(self.harness.charm._WORDPRESS_DEFAULT_PLUGINS + ["123"]),
             "removing plugins from plugins config should trigger plugin deletion"
         )
+
+    def _standard_plugin_test(
+            self, plugin, plugin_config, excepted_options, additional_check_after_install=None
+    ):
+        plugin_config_keys = list(plugin_config.keys())
+        self._setup_replica_consensus()
+        db_config = {
+            "db_host": "config_db_host",
+            "db_name": "config_db_name",
+            "db_user": "config_db_user",
+            "db_password": "config_db_password",
+        }
+        self.patch.allow_database(db_config)
+        self.harness.update_config(db_config)
+
+        self.harness.update_config(plugin_config)
+
+        self.assertEqual(
+            self.patch.get_active_plugins(db_host="config_db_host", db_name="config_db_name"),
+            {plugin},
+            f"{plugin} should be activated after {plugin_config_keys} being set"
+        )
+        self.assertEqual(
+            self.patch.get_options(db_host="config_db_host", db_name="config_db_name"),
+            excepted_options,
+            "akismet options should be set correctly"
```

Comment:
Should this be explicitly referencing akismet? I think it should be whichever plugin is being used here, right?

Reply:
Fixed, thanks!